### ISSUE TITLE
[Please read] ssbjenkins oversubscription

### ISSUE THREAD

**Issue by jhunkeler** [spacetelescope/hlapipeline#65]
`ssbjenkins` has been "jammed" for days with `hlapipeline` tests that never complete. Out of curiosity I ran the test I suspected was hanging things up on our end...

```
$ pytest -v -s --bigdata tests/test_bigRandom.py
# ...
==== 1 failed, 6619 warnings in 27678.56 seconds ====
```

Please improve, disable, mark `--slow`, or remove this, and any other tests that cannot be completed in a reasonable amount of time at your earliest convenience. The `Jenkinsfile` configuration spawns more jobs than we have executors, and all of them are running the same test suite. No other `spacetelescope/*` jobs can run when `hlapipeline` receives a commit.

`ssbjenkins` is not a regression testing platform, so please limit your testing to unit tests, fast end-to-end tests, and noninvasive smoke-tests. If this project requires `--slow` tests to be executed on a regular schedule we will be more than happy to try to find a server better equipped to handle the load.

**Comment by stscicrawford** [comment]
If you haven't already fixed it, I believe you can kill all the hlapipeline jenkins tests.   This should be fixed in #67

**Comment by mdlpstsci** [comment]
VERY SORRY.  I had the decorator slow commented out as I am running the tests on my desktop.  I have fixed this and it is awaiting approval in spacetelescope/hlapipeline PR# 67.

**Comment by stscicrawford** [comment]
Closed by #67
